dashboard/app/jobs.go

```diff
@@ -1145,6 +1143,23 @@ func loadPendingJob(c context.Context, managers map[string]dashapi.ManagerJobs)
 	return nil, nil, nil
 }
 
+// activeManager determines the manager currently responsible for all bugs found by
+// the specified manager.
+func activeManager(manager string) (string, *ConfigManager) {
+	for _, ns := range config.Namespaces {
```

Comment:
We don't enforce that manager names are globally unique. So I think we need to pass bug+crash (or bug+manager) to this functions and use bug.Namespace.

Reply:
I've changed the function to accept a manager + namespace.